### PR TITLE
Structural mechanics/fix inertia in beam element

### PR DESCRIPTION
**Pull request by AndreasWinterstein**
This PR fixes a bug in the 3D beam element. The moment of inertia going into the mass matrix is implemented correctly. If there is no user input is is assumed to be Iy + Iz as proposed in literature. Otherwise the value is used from the input.

### CODE REVIEW

**Comment by AndreasWinterstein** [comment]
@KlausBSautter this was not tested. Thus, we should think about creating a test for this.

**Review by KlausBSautter — DISMISSED** [review]
thanks !

File: applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_3D2N.cpp [hunk 1]
```diff
@@ -1395,9 +1395,17 @@ void CrBeamElement3D2N::CalculateConsistentMassMatrix(
     const double A = GetProperties()[CROSS_AREA];
     const double E = GetProperties()[YOUNG_MODULUS];
 
-    const double J = GetProperties()[TORSIONAL_INERTIA];
```
Comment (philbucher) [inline applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_3D2N.cpp:1398]: this is no longer used?

Comment (AndreasWinterstein) [reply applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_3D2N.cpp:1398]: In the mass matrix not.

**Review by KlausBSautter — APPROVED** [review]

**Pull request merged**